### PR TITLE
Added QR scanning

### PR DESCRIPTION
**Pull request by jameskokoska**

### CODE REVIEW

**Comment by jameskokoska** [comment]
https://user-images.githubusercontent.com/50821962/173501287-3590d613-397b-47e5-8cab-95b1b0dbc199.mp4

**Comment by jameskokoska** [comment]
https://user-images.githubusercontent.com/50821962/173501456-59a754aa-5ccf-4225-8822-3ada8576dc23.mp4

File: client/src/pages/Profile/functions.jsx [hunk 1]
```diff
@@ -94,6 +94,62 @@ export function getDaysFroshSchedule() {
...
+      name: undefined,
+      pronouns: undefined,
+      shirtSize: undefined,
+      froshGroup: undefined,
+      discipline: undefined,
+    };
+  }
+}
+
 export function getQRCodeString() {
```
[the remark this answers is not in the document]
Comment (jameskokoska) [reply client/src/pages/Profile/functions.jsx:120]: https://github.com/UofT-Frosh-Orientation/orientation-website/issues/148

**Review by natapokie — APPROVED** [review]
the page looks great and the qr scanner is really cool !!! 🤩🤩

**Comment by Cali0707** [comment]
I think this PR should be good to go @jameskokoska

**Review by Cali0707 — APPROVED** [review]

**Pull request merged**